[2024-10-15T22:10:46.588+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-15T22:10:46.604+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: LoadWebScraping.spark_submit_task manual__2024-10-15T22:10:45.344514+00:00 [queued]>
[2024-10-15T22:10:46.613+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: LoadWebScraping.spark_submit_task manual__2024-10-15T22:10:45.344514+00:00 [queued]>
[2024-10-15T22:10:46.614+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-15T22:10:46.627+0000] {taskinstance.py:2888} INFO - Executing <Task(SparkSubmitOperator): spark_submit_task> on 2024-10-15 22:10:45.344514+00:00
[2024-10-15T22:10:46.633+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3903) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-15T22:10:46.635+0000] {standard_task_runner.py:72} INFO - Started process 3905 to run task
[2024-10-15T22:10:46.635+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'LoadWebScraping', 'spark_submit_task', 'manual__2024-10-15T22:10:45.344514+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/LoadWebScraping.py', '--cfg-path', '/tmp/tmp9l_btgq8']
[2024-10-15T22:10:46.638+0000] {standard_task_runner.py:105} INFO - Job 62: Subtask spark_submit_task
[2024-10-15T22:10:46.687+0000] {task_command.py:467} INFO - Running <TaskInstance: LoadWebScraping.spark_submit_task manual__2024-10-15T22:10:45.344514+00:00 [running]> on host eeedff29a94a
[2024-10-15T22:10:46.935+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='LoadWebScraping' AIRFLOW_CTX_TASK_ID='spark_submit_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T22:10:45.344514+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-15T22:10:45.344514+00:00'
[2024-10-15T22:10:46.936+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-15T22:10:46.956+0000] {base.py:84} INFO - Retrieving connection 'spark_default'
[2024-10-15T22:10:46.957+0000] {spark_submit.py:327} ERROR - Failed to decode base64 keytab: Incorrect padding
[2024-10-15T22:10:46.957+0000] {spark_submit.py:304} INFO - Could not load connection string spark_default, defaulting to spark://spark:7077
[2024-10-15T22:10:46.959+0000] {spark_submit.py:473} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --driver-class-path /opt/bitnami/spark/jars/postgresql-42.5.0.jar --jars /opt/bitnami/spark/jars/postgresql-42.5.0.jar --name arrow-spark --deploy-mode client /opt/bitnami/spark/src/scripts_spark/LoadFileToDB.py
[2024-10-15T22:10:48.626+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:48 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-10-15T22:10:48.822+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:48 WARN DependencyUtils: Local jar /opt/bitnami/spark/jars/postgresql-42.5.0.jar does not exist, skipping.
[2024-10-15T22:10:50.129+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SparkContext: Running Spark version 3.5.2
[2024-10-15T22:10:50.130+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SparkContext: OS info Linux, 5.15.153.1-microsoft-standard-WSL2, amd64
[2024-10-15T22:10:50.131+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SparkContext: Java version 17.0.12
[2024-10-15T22:10:50.159+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO ResourceUtils: ==============================================================
[2024-10-15T22:10:50.160+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-10-15T22:10:50.160+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO ResourceUtils: ==============================================================
[2024-10-15T22:10:50.161+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SparkContext: Submitted application: Test PostgreSQL Connection
[2024-10-15T22:10:50.181+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-10-15T22:10:50.189+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO ResourceProfile: Limiting resource is cpu
[2024-10-15T22:10:50.190+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-10-15T22:10:50.231+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SecurityManager: Changing view acls to: ***
[2024-10-15T22:10:50.232+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SecurityManager: Changing modify acls to: ***
[2024-10-15T22:10:50.233+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SecurityManager: Changing view acls groups to:
[2024-10-15T22:10:50.233+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SecurityManager: Changing modify acls groups to:
[2024-10-15T22:10:50.234+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-10-15T22:10:50.456+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO Utils: Successfully started service 'sparkDriver' on port 46527.
[2024-10-15T22:10:50.485+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SparkEnv: Registering MapOutputTracker
[2024-10-15T22:10:50.517+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SparkEnv: Registering BlockManagerMaster
[2024-10-15T22:10:50.531+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-10-15T22:10:50.532+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-10-15T22:10:50.536+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-10-15T22:10:50.553+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-89b329ff-01dc-447a-a53f-2cf1582e3b82
[2024-10-15T22:10:50.565+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-10-15T22:10:50.578+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-10-15T22:10:50.745+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-10-15T22:10:50.823+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-10-15T22:10:50.857+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 ERROR SparkContext: Failed to add /opt/bitnami/spark/jars/postgresql-42.5.0.jar to Spark environment
[2024-10-15T22:10:50.858+0000] {spark_submit.py:649} INFO - java.io.FileNotFoundException: Jar /opt/bitnami/spark/jars/postgresql-42.5.0.jar not found
[2024-10-15T22:10:50.858+0000] {spark_submit.py:649} INFO - at org.apache.spark.SparkContext.addLocalJarFile$1(SparkContext.scala:2095)
[2024-10-15T22:10:50.859+0000] {spark_submit.py:649} INFO - at org.apache.spark.SparkContext.addJar(SparkContext.scala:2151)
[2024-10-15T22:10:50.859+0000] {spark_submit.py:649} INFO - at org.apache.spark.SparkContext.$anonfun$new$15(SparkContext.scala:521)
[2024-10-15T22:10:50.860+0000] {spark_submit.py:649} INFO - at org.apache.spark.SparkContext.$anonfun$new$15$adapted(SparkContext.scala:521)
[2024-10-15T22:10:50.860+0000] {spark_submit.py:649} INFO - at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
[2024-10-15T22:10:50.861+0000] {spark_submit.py:649} INFO - at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
[2024-10-15T22:10:50.861+0000] {spark_submit.py:649} INFO - at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
[2024-10-15T22:10:50.862+0000] {spark_submit.py:649} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:521)
[2024-10-15T22:10:50.862+0000] {spark_submit.py:649} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2024-10-15T22:10:50.862+0000] {spark_submit.py:649} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-10-15T22:10:50.863+0000] {spark_submit.py:649} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2024-10-15T22:10:50.863+0000] {spark_submit.py:649} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-10-15T22:10:50.864+0000] {spark_submit.py:649} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2024-10-15T22:10:50.864+0000] {spark_submit.py:649} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2024-10-15T22:10:50.865+0000] {spark_submit.py:649} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2024-10-15T22:10:50.866+0000] {spark_submit.py:649} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-10-15T22:10:50.867+0000] {spark_submit.py:649} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2024-10-15T22:10:50.868+0000] {spark_submit.py:649} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2024-10-15T22:10:50.868+0000] {spark_submit.py:649} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2024-10-15T22:10:50.869+0000] {spark_submit.py:649} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-10-15T22:10:50.869+0000] {spark_submit.py:649} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-10-15T22:10:50.869+0000] {spark_submit.py:649} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-10-15T22:10:50.943+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2024-10-15T22:10:50.988+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:50 INFO TransportClientFactory: Successfully created connection to spark/172.18.0.4:7077 after 26 ms (0 ms spent in bootstraps)
[2024-10-15T22:10:51.129+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20241015221051-0010
[2024-10-15T22:10:51.136+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20241015221051-0010/0 on worker-20241015211326-172.18.0.2-36469 (172.18.0.2:36469) with 6 core(s)
[2024-10-15T22:10:51.141+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO StandaloneSchedulerBackend: Granted executor ID app-20241015221051-0010/0 on hostPort 172.18.0.2:36469 with 6 core(s), 1024.0 MiB RAM
[2024-10-15T22:10:51.153+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 33347.
[2024-10-15T22:10:51.154+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO NettyBlockTransferService: Server created on eeedff29a94a:33347
[2024-10-15T22:10:51.155+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-10-15T22:10:51.162+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, eeedff29a94a, 33347, None)
[2024-10-15T22:10:51.166+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO BlockManagerMasterEndpoint: Registering block manager eeedff29a94a:33347 with 434.4 MiB RAM, BlockManagerId(driver, eeedff29a94a, 33347, None)
[2024-10-15T22:10:51.169+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, eeedff29a94a, 33347, None)
[2024-10-15T22:10:51.170+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, eeedff29a94a, 33347, None)
[2024-10-15T22:10:51.191+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20241015221051-0010/0 is now RUNNING
[2024-10-15T22:10:51.397+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-10-15T22:10:51.575+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-10-15T22:10:51.577+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:51 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2024-10-15T22:10:52.357+0000] {spark_submit.py:649} INFO - Traceback (most recent call last):
[2024-10-15T22:10:52.358+0000] {spark_submit.py:649} INFO - File "/opt/bitnami/spark/src/scripts_spark/LoadFileToDB.py", line 21, in <module>
[2024-10-15T22:10:52.361+0000] {spark_submit.py:649} INFO - .load()
[2024-10-15T22:10:52.362+0000] {spark_submit.py:649} INFO - ^^^^^^
[2024-10-15T22:10:52.366+0000] {spark_submit.py:649} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/readwriter.py", line 314, in load
[2024-10-15T22:10:52.372+0000] {spark_submit.py:649} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2024-10-15T22:10:52.376+0000] {spark_submit.py:649} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 179, in deco
[2024-10-15T22:10:52.379+0000] {spark_submit.py:649} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-10-15T22:10:52.397+0000] {spark_submit.py:649} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o33.load.
[2024-10-15T22:10:52.398+0000] {spark_submit.py:649} INFO - : java.sql.SQLException: No suitable driver
[2024-10-15T22:10:52.401+0000] {spark_submit.py:649} INFO - at java.sql/java.sql.DriverManager.getDriver(DriverManager.java:299)
[2024-10-15T22:10:52.402+0000] {spark_submit.py:649} INFO - at org.apache.spark.sql.execution.datasources.jdbc.JDBCOptions.$anonfun$driverClass$2(JDBCOptions.scala:109)
[2024-10-15T22:10:52.403+0000] {spark_submit.py:649} INFO - at scala.Option.getOrElse(Option.scala:189)
[2024-10-15T22:10:52.404+0000] {spark_submit.py:649} INFO - at org.apache.spark.sql.execution.datasources.jdbc.JDBCOptions.<init>(JDBCOptions.scala:109)
[2024-10-15T22:10:52.405+0000] {spark_submit.py:649} INFO - at org.apache.spark.sql.execution.datasources.jdbc.JDBCOptions.<init>(JDBCOptions.scala:41)
[2024-10-15T22:10:52.407+0000] {spark_submit.py:649} INFO - at org.apache.spark.sql.execution.datasources.jdbc.JdbcRelationProvider.createRelation(JdbcRelationProvider.scala:34)
[2024-10-15T22:10:52.408+0000] {spark_submit.py:649} INFO - at org.apache.spark.sql.execution.datasources.DataSource.resolveRelation(DataSource.scala:346)
[2024-10-15T22:10:52.409+0000] {spark_submit.py:649} INFO - at org.apache.spark.sql.DataFrameReader.loadV1Source(DataFrameReader.scala:229)
[2024-10-15T22:10:52.409+0000] {spark_submit.py:649} INFO - at org.apache.spark.sql.DataFrameReader.$anonfun$load$2(DataFrameReader.scala:211)
[2024-10-15T22:10:52.410+0000] {spark_submit.py:649} INFO - at scala.Option.getOrElse(Option.scala:189)
[2024-10-15T22:10:52.410+0000] {spark_submit.py:649} INFO - at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:211)
[2024-10-15T22:10:52.411+0000] {spark_submit.py:649} INFO - at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:172)
[2024-10-15T22:10:52.411+0000] {spark_submit.py:649} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-10-15T22:10:52.412+0000] {spark_submit.py:649} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-10-15T22:10:52.412+0000] {spark_submit.py:649} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-10-15T22:10:52.412+0000] {spark_submit.py:649} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:569)
[2024-10-15T22:10:52.413+0000] {spark_submit.py:649} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-10-15T22:10:52.413+0000] {spark_submit.py:649} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-10-15T22:10:52.414+0000] {spark_submit.py:649} INFO - at py4j.Gateway.invoke(Gateway.java:282)
[2024-10-15T22:10:52.414+0000] {spark_submit.py:649} INFO - at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-10-15T22:10:52.415+0000] {spark_submit.py:649} INFO - at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-10-15T22:10:52.415+0000] {spark_submit.py:649} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-10-15T22:10:52.416+0000] {spark_submit.py:649} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-10-15T22:10:52.416+0000] {spark_submit.py:649} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-10-15T22:10:52.417+0000] {spark_submit.py:649} INFO - 
[2024-10-15T22:10:52.462+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO SparkContext: Invoking stop() from shutdown hook
[2024-10-15T22:10:52.462+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-10-15T22:10:52.471+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO SparkUI: Stopped Spark web UI at http://eeedff29a94a:4040
[2024-10-15T22:10:52.475+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-10-15T22:10:52.478+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-10-15T22:10:52.503+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-10-15T22:10:52.515+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO MemoryStore: MemoryStore cleared
[2024-10-15T22:10:52.517+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO BlockManager: BlockManager stopped
[2024-10-15T22:10:52.522+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-10-15T22:10:52.525+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-10-15T22:10:52.535+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO SparkContext: Successfully stopped SparkContext
[2024-10-15T22:10:52.536+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO ShutdownHookManager: Shutdown hook called
[2024-10-15T22:10:52.536+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO ShutdownHookManager: Deleting directory /tmp/spark-75253daa-47b7-4fee-b8a6-6ff86b27c10f
[2024-10-15T22:10:52.541+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO ShutdownHookManager: Deleting directory /tmp/spark-75253daa-47b7-4fee-b8a6-6ff86b27c10f/pyspark-60be3e22-a51e-48c8-a3ce-5f8140670254
[2024-10-15T22:10:52.544+0000] {spark_submit.py:649} INFO - 24/10/15 22:10:52 INFO ShutdownHookManager: Deleting directory /tmp/spark-f3d515cc-5e89-444a-a3bf-6c8fdae3a3e2
[2024-10-15T22:10:52.576+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 176, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 575, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --driver-class-path /opt/bitnami/spark/jars/postgresql-42.5.0.jar --jars /opt/bitnami/spark/jars/postgresql-42.5.0.jar --name arrow-spark --deploy-mode client /opt/bitnami/spark/src/scripts_spark/LoadFileToDB.py. Error code is: 1.
[2024-10-15T22:10:52.579+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=LoadWebScraping, task_id=spark_submit_task, run_id=manual__2024-10-15T22:10:45.344514+00:00, execution_date=20241015T221045, start_date=20241015T221046, end_date=20241015T221052
[2024-10-15T22:10:52.591+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-15T22:10:52.592+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 62 for task spark_submit_task (Cannot execute: spark-submit --master spark://spark:7077 --driver-class-path /opt/bitnami/spark/jars/postgresql-42.5.0.jar --jars /opt/bitnami/spark/jars/postgresql-42.5.0.jar --name arrow-spark --deploy-mode client /opt/bitnami/spark/src/scripts_spark/LoadFileToDB.py. Error code is: 1.; 3905)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 176, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 575, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --driver-class-path /opt/bitnami/spark/jars/postgresql-42.5.0.jar --jars /opt/bitnami/spark/jars/postgresql-42.5.0.jar --name arrow-spark --deploy-mode client /opt/bitnami/spark/src/scripts_spark/LoadFileToDB.py. Error code is: 1.
[2024-10-15T22:10:52.611+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-15T22:10:52.623+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-15T22:10:52.628+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
